xwiki-commons-tools/xwiki-commons-tool-webjar-node-handlers/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ *
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ *
+-->
+
+<!-- TODO:
+* missing LICENSE file
+* missing NOTICE file
+* missing copy of variables.less for xwiki-platform-livedata-webjars
+-->
+
+<component-set>
+  <components>
+    <component>
+      <role>org.apache.maven.lifecycle.mapping.LifecycleMapping</role>
+      <role-hint>webjar-node</role-hint>
+      <implementation>org.apache.maven.lifecycle.mapping.DefaultLifecycleMapping</implementation>
+      <configuration>
+        <phases>
+          <validate>
+            <mojos>
+              <mojo>
+                <goal>org.apache.maven.plugins:maven-resources-plugin:copy-resources</goal>
+                <configuration>
+                  <outputDirectory>${project.build.directory}/vue</outputDirectory>
+                  <resources>
+                    <resource>
+                      <directory>src/main/vue</directory>
+                      <excludes>
+                        <exclude>dist</exclude>
+                        <exclude>node_modules</exclude>
+                      </excludes>
+                    </resource>
+                  </resources>
+                </configuration>
+              </mojo>
+            </mojos>
+          </validate>
+          <process>
+            <mojos>
+              <mojo>
+                <goal>org.xwiki.commons:xwiki-commons-tool-remote-resource-plugin</goal>
+                <configuration>
+                  <projectsData>LICENSES</projectsData>
+                  <resourceBundles>
+                    <resourceBundle>org.xwiki.commons:xwiki-commons-tool-license-resources:${commons.version}
```

Comment:
Based on https://github.com/xwiki/xwiki-commons/blob/master/xwiki-commons-pom/pom.xml#L91-L104 but I'm doing something wrong since the LICENSE and NOTICE files are still missing on the produced jar.